enhancements/bare-metal-fulfillment/README.md

```diff
@@ -0,0 +1,267 @@
+---
+title: bare-metal-fulfillment
+authors:
+  - Tzu-Mainn Chen
+creation-date: 2025-09-08
+last-updated: 2025-09-08
+tracking-link:
+  - None
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Bare Metal Fulfillment
+
+## Summary
+
+Bare metal fulfillment refers to the process through which a tenant acquires bare metal hosts and configures their networking.
+Afterwards, the tenant will be able to perform selected operations on the host: inventory information, power control, and
+serial console access. Any further operations - for example, image-based provisioning - are outside the scope of the O-SAC
+solution and must be integrated by the cloud provider.
+
+In this context, a “tenant” is a person with admin rights to the requested bare metal resources; at the MOC, this person would be
+a ColdFront PI or manager.
+
+## Motivation
+
+Bare metal fulfillment is a fundamental requirement for O-SAC, identified as an explicit need for the MOC. It will also serve
+as the basis of bare metal configuration for other fulfillment workflows, such as OpenShift cluster fulfillment.
+
+### User Stories
+
+* As a tenant, I want to be able to see available bare metal resource classes.
+* As a tenant, I want to be able to select hosts for a bare metal cluster by resource class.
+* As a tenant, I want to be able to view inventory details for my hosts.
+* As a tenant, I want to be able to connect to the serial console of my hosts.
+* As a tenant, I want to be able to perform basic power control (power on/power off/reset) of my hosts.
+* As a tenant, I want to be able to attach hosts to existing networks.
+* As a tenant, I want to be able to modify the network connectivity of my hosts by attaching or detaching networks.
+* As a tenant, I want to be able to manage connectivity at the interface level in order to support hardware that may have both regular and high performance interfaces (for example, some hosts may have high-bandwidth connectivity for GPU workloads, and it may be necessary to attach specific networks to these interfaces).
+
+### Goals
+
+We will implement bare metal fulfillment in the O-SAC solution, covering the user stories described above. This implementation will
+be first deployed and tested in the MOC; as a result, the implementation will rely upon ESI for bare metal management (although
+it will be designed to allow easy replacement of the bare metal management layer).
```

Comment:
```suggestion
it will be designed to allow easy replacement of the bare metal management layer). While not addressed in this proposal, a long term goal of bare metal fulfillment is to have it be used as a basis of cluster fulfillment.
```